src/pal/inc/pal.h

```diff
@@ -262,8 +262,8 @@ typedef char * va_list;
 #define UINT_MAX      0xffffffff
 
 #define LONG_MIN    (-2147483647L - 1)
-#define LONG_MAX      2147483647L
-#define ULONG_MAX     0xffffffffUL
+//#define LONG_MAX      2147483647L
+//#define ULONG_MAX     0xffffffffUL
```

Comment:
@jkotas Should I delete them, right?

Reply:
Yes, you can delete as much as necessary - as long as everything still builds and works.